### ISSUE TITLE
Spacer does not display using show on 0.12.11

### ISSUE THREAD

**Issue by jonalm** [bokeh/bokeh#7266]
Hi there,

I didn't find any mentions of `Spacer` in the changelog, so this might be bug?

I run bokeh using jupyter notebook 5.2.1, python 3.6.3 on OSX.

The following code works fine under 0.12.10 but does not manage to display correctly under 0.12.11.

```python
from bokeh.io import show, output_notebook
from bokeh.plotting import figure
from bokeh.models import Line, Plot, ColumnDataSource, Range1d
from bokeh.layouts import column, row, Spacer

output_notebook()

xrange = Range1d(0,1)
source = ColumnDataSource(dict(x=[0,1],y1=[0,1],y2=[1,0]))
plot_1 = Plot(width=100, height=100, x_range=xrange, y_range=xrange)
plot_2 = Plot(width=100, height=100, x_range=xrange, y_range=xrange)
plot_1.add_glyph(source, Line(x='x',y='y1'))
plot_2.add_glyph(source, Line(x='x',y='y2'))

show(column(plot_1, Spacer(height=10), plot_2))
```
Display under 0.12.10

![screenshot 2017-12-01 08 48 39](https://user-images.githubusercontent.com/5505854/33473198-bb0ea6a0-d675-11e7-99c7-832db864ba6f.png)

Display under 0.12.11

![screenshot 2017-12-01 08 49 25](https://user-images.githubusercontent.com/5505854/33473212-cbf720be-d675-11e7-8b31-31cefc6b31d2.png)

**Comment by bryevdv** [comment]
ping @mattpap

**Comment by dantamont** [comment]
I'd like to confirm that I am also having issues with spacers not working properly.

**Comment by ivan-usov** [comment]
As a workaround, one can specify a missing height/width parameter and set it to 0, e.g.
`Spacer(height=10, width=0)`

**Comment by mattpap** [comment]
I'm pretty sure this was fixed as a side effect of other work in current `master`, but this needs to be confirmed.

**Comment by ivan-usov** [comment]
Indeed, it's been fixed at least in `bokeh/0.12.14dev3`:
![selection_105](https://user-images.githubusercontent.com/13196195/35101428-cfb98c36-fc5f-11e7-9538-2544c2d968d4.png)

**Comment by bryevdv** [comment]
Well that's always a nice surprise. Closing then, but @mattpap if you happen to know the PR that was relevant to this getting fixed it would be good to link it as a reference.
